app/models/warehouse.rb

```diff
@@ -0,0 +1,2 @@
+class Warehouse < ApplicationRecord
+end
```

Comment:
I don't know if is in the requirements, but you could add some `rules` in the model, for example, set some columns as required, this help you to avoid save empty records, you could check that [here](https://guides.rubyonrails.org/active_record_validations.html) 😄

Reply:
if you add validation please add some specs of the model 👀